ansible/files/bootstrap/telemetry.sh

```diff
@@ -152,6 +152,12 @@ bash /mnt/$ANACONDA_SCRIPT -b -p $ANACONDA_PATH
 PIP_REQUIREMENTS_FILE=/tmp/requirements.txt
 aws s3 cp $TELEMETRY_CONF_BUCKET/bootstrap/python-requirements.txt $PIP_REQUIREMENTS_FILE
 $ANACONDA_PATH/bin/pip install -r $PIP_REQUIREMENTS_FILE
+
+conda create -n zeppelin python=3.6 cairo pillow -y -q
+source activate zeppelin
+pip install python-zeppelin
```

Comment:
My vote is for a python-3 requirements file, and ideally setup pyup to update those dependencies as well.

@comloo if you'd like to handle the transition of everything to Python 3.x, we're happy to let you take the reigns :)